### DISCUSSION TITLE
Sorting of logs

### DISCUSSION THREAD

**Discussion by jia2332** [apache/skywalking#8407] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

SkyWalking UI
Maybe you can consider using `%lsn` `LocalSequenceNumberConverter.class` to solve the problem of log sorting?

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
What do you mean by using `LocalSequenceNumberConverter`? There is no such thing in log reporting. SkyWalking accepts logs from many different agents and 3rd party tools.

**Reply by jia2332** [reply]
Because the log order is different from that of console output
![企业微信截图_20220112115222](https://user-images.githubusercontent.com/24456816/149072596-abe03fd8-a6d3-4820-9adf-173971a094cb.png)
![企业微信截图_20220112115444](https://user-images.githubusercontent.com/24456816/149072603-ae70000d-0662-4d45-a7cb-36d6a8841ab4.png)
I don't know if elasticsearch or H2 can't be sorted correctly because of the millisecond level logs.
Therefore, it is recommended that the collation use timestamp and local serial number, or use microsecond level

**Reply by wu-sheng** [reply]
`ms` is a global-level definition of time precision. And SkyWalking is an APM running for product env, which is impossible to print logs in this high frequency. This kind of log output policy will kill the online system.

If(just if) we set up time precision to `ns`, it will increase the load and storage cost of ES cluster and network.
